src/Class6.java

```diff
@@ -0,0 +1,6 @@
+public class Class6 {
+    public static void main(String[] args) {
+        System.out.println("urjujr");
```

Comment:
some text
```suggestion
bla-bla-bla
```
another text